build.sh

```diff
@@ -90,15 +125,17 @@ if [ -e zephyr/module.yml ]; then
     >&2 echo "ERROR: Branch not implemented!"; exit 1
 fi
 
+header "Building All the things!"
+
+build_init
+
+sh ./patch_art.sh
```

Comment:
Should this be part of `build_init`?

Reply:
My understanding is that `build_init` sets up the build, e.g. directories and all, and that `patch_art` is already one of the steps of the build.